src/components/mailchimpbutton.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { Component, ReactElement } from 'react';
+import MailchimpSubscribe from 'react-mailchimp-subscribe';
+import { withAlert } from 'react-alert';
+import getServerURL from '../serverOverride';
+import { isValidEmail } from '../lib/Validations/Validations';
+import CustomFormButton from './mailChimpSubscribe/CustomFormButton';
+
+interface Props {
+  alert: any;
+}
+
+interface State {
+  email: string;
+  buttonState: string;
+  emailValidator: string;
+  submitButtonListener: boolean;
+}
+
+class Mailchimpbutton extends Component<Props, State, {}> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      email: '',
+      buttonState: '',
+      emailValidator: '',
+      submitButtonListener: false,
+    };
+  }
+
+  componentDidMount() {
+    window.scrollTo(0, 0);
+  }
+
+  clearInput = (): void => {
+    this.setState({
+      email: '',
+      buttonState: '',
+      emailValidator: '',
+    });
+  };
+
+  validateEmail = async (): Promise<void> => {
+    const { email } = this.state;
+    if (isValidEmail(email)) {
+      await new Promise((resolve) => this.setState({ emailValidator: 'true' }, resolve));
+    } else {
+      await new Promise((resolve) => this.setState({ emailValidator: 'false' }, resolve));
+    }
+  };
+
+  emailMessage = (): ReactElement<{}> => {
+    const { emailValidator } = this.state;
+    if (emailValidator === 'true') {
+      return <div className="valid-feedback" />;
+    }
+    if (emailValidator === 'false') {
+      return (
+        <div className="invalid-feedback text-left">
+          Please enter a valid email address
+        </div>
+      );
+    }
+    return <div />;
+  };
+
+  colorToggle = (inputString: string): string => {
+    if (inputString === 'true') {
+      return 'is-valid';
+    }
+    if (inputString === 'false') {
+      return 'is-invalid';
+    }
+    return '';
+  };
+
+  handleSubmitWithRecaptcha = async (event: any) => {
```

Comment:
delete this whole function as it isn't being used

Reply:
The function is being used for the sign-up button on line 161. The purpose of the function is to click on the signup button and display a confirmation message or an error message. If we remove this function we would not be able to get an alert telling us if the email successfully went through.